features/0249-rich-schema-contexts/README.md

```diff
@@ -10,10 +10,14 @@
 ## Summary
 [summary]: #summary
 
-Every rich schema object has an associated `@context`. Contexts are JSON-LD
+Every rich schema object may have an associated `@context`. Contexts are JSON or JSON-LD
```

Comment:
How will it be known if it's a JSON vs JSON-LD object? For example, if the context is anchored in JSON-LD form, then it's acceptable that the context is IRIs that are expanded by the document loader. In the case of a JSON consumer though, I'm not certain how they would receive the necessary detail to handle processing of an IRI. Could we provide additional details on this?

Reply:
Any Rich Schema (which is a JSON-LD object) may reference Contexts by their `id`. The Context's content can be used then for substituting the attributes with IRIs within the Rich Schema object.

The question is whether the Context's content itself must be a JSON-LD as well. 
As I can see from https://json-ld.org/spec/latest/json-ld/#the-context, it's not.
And as I can see from the example of Contexts that we have (@brentzundel can correct me if I'm wrong),  the only requirement for the Context's format is to have a `@context` property containing a map of terms to IRIs. A Context may not have `@id` and `@type` properties expecting from any JSON_LD object.

Reply:
My understanding of the `@context` objects is that simple ones are JSON objects, but there may be more advanced options that require JSON-LD. 
It may be a moot question though, because the object that **has** a `@context` is a JSON-LD object and and processing that object will include dealing with the `@context`. I think its safe to assume that a JSON-LD processor will know how to deal with `@context` objects.

Reply:
Ahh good points. Thanks for the clarifications